pages/pagebase.css

```diff
@@ -19,15 +19,18 @@
 	h1 {
 		font-size: 2.75em;
 		font-weight: normal;
-		margin-bottom: 0.25em;
+		margin-bottom: 0.45em;
 		margin-top: 0;
 	}
 	h2 {
```

Comment:
This text is kind of hard to read. Could this have less opacity, or be a little bit larger?